ArrayOps.java

```diff
@@ -1,26 +1,173 @@
+            {
+                max=array[i];
+                tempI=i;
+            }
+        }
+
+        int max1=array[0];
+        for(int j=1;j<array.length;j++)
+        {
+            
+            if(array[j]>max1 && tempI!=j) //&& array[j]!=max
+            {
+                max1=array[j];
+            }
+            else
+            {
+                if(array[j]==max && j!=tempI)
+                {
+                    max1=array[j];
+                    j=array.length; //break;
+                }
+            }
+        }
+        return max1;
     }
 
     public static boolean containsTheSameElements(int [] array1,int [] array2) {
         // Write your code here:
-        return false;
+        boolean sameElements1=true;
+        boolean sameElements2=true;
+        
+            for(int i=0; i<array1.length; i++)
+            {
+                for(int j=0; j<array2.length; j++)
+                {
+                    if(array1[i]==array2[j])
+                    {
+                        sameElements1=true;
+                        j=array2.length;
+                    }
+                    else
+                    {
+                        sameElements1=false;
+                    }
+                }
+            
+                if(sameElements1==false)
+                {
+                    break;
+                }
+            }
+        
+        
+            for(int i=0; i<array2.length; i++)
+            {
+                for(int j=0; j<array1.length; j++)
+                {
+                    if(array2[i]==array1[j])
+                    {
+                        sameElements2=true;
+                        j=array1.length;
+                    }
+                    else
+                    {
+                        sameElements2=false;
+                    }
+                }
+            
+                if(sameElements2==false)
+                {
+                    break;
+                }
+            }
+        
+
+        
+        return sameElements1&&sameElements2;
     }
 
     public static boolean isSorted(int [] array) {
         // Write your code here:
-        return false;
+        boolean sorted=true;
+        for(int i=1; i<array.length;i++)
+        {
+            if(array[i]==array[i-1])
+            {  
+                sorted=false;
+            }
+            else
+            {
+                sorted=true;
+                if(array[i]>array[i-1])
+                {
+                    for(int j=i+1;j<array.length;j++)
+                    {
+                        if(array[j]>=array[j-1]) //&&array
+                        {
+                            sorted=true;
+                        }
+                        else
+                        {
+                          sorted=false;
+                          j=array.length;//break;
+                          i=array.length;
+                        }
+                    }
+                }
```

Comment:
why do you need another loop? It cause checking if the array increasing\decreasing `array.length` times while you need to check it only once...